pkg/p2p/libp2p/internal/handshake/handshake.go

```diff
@@ -162,7 +162,7 @@ func (s *Service) Handshake(ctx context.Context, stream p2p.Stream, peerMultiadd
 			Signature: bzzAddress.Signature,
 		},
 		NetworkID:      s.networkID,
-		Light:          s.lightNode,
+		Light:          !s.fullNode,
```

Comment:
Is the field name kept `Light` just for the sake of no change? maybe better just to change now that there are breaking changes

Reply:
yes, it was already like that. I can change it to `FullNode` @acud what do you think?

Reply:
no particular opinion here tbh

Reply:
up to you :+1: